src/engines/Cesium/common.ts

```diff
@@ -554,7 +554,7 @@ export const colorBlendModeFor3DTile = (
       replace: Cesium3DTileColorBlendMode.REPLACE,
       mix: Cesium3DTileColorBlendMode.MIX,
     }) as { [key in string]?: Cesium3DTileColorBlendMode }
-  )[colorBlendMode || ""];
+  )[colorBlendMode || ""] || Cesium3DTileColorBlendMode.HIGHLIGHT;
```

Comment:
I think you can use default parameter like below.
```
export const colorBlendModeFor3DTile = (
  colorBlendMode?: "highlight" | "replace" | "mix" | "default" = "default",
) => ...
```

Reply:
Thanks. I have updated this. `colorBlendMode` might not be an optional parameter after giving it a initialized value.
<img width="720" alt="Screenshot 2024-11-28 at 7 50 46 AM" src="https://github.com/user-attachments/assets/fe5fc1ca-3883-4c50-85ff-cf25ba9281e7">

Reply:
Yes, you can remove `?`.

Reply:
Yes, you can remove `?`.

Reply:
Thanks. I've already removed the `?`